pkg/utils/report/labels.go

```diff
@@ -66,6 +67,17 @@ func PolicyLabel(policy kyvernov1.PolicyInterface) string {
 	return PolicyLabelPrefix(policy) + policy.GetName()
 }
 
+func CleanupKyvernoLabels(obj metav1.Object) {
+	labels := obj.GetLabels()
+	for key := range labels {
+		if strings.Contains(key, LabelDomain) {
+			delete(labels, key)
+		}
+	}
+
+	obj.SetLabels(labels)
```

Comment:
do we need to set labels ?

Reply:
Probably not because the map is not cloned and will be mutated anyway. Just wanted to be explicit.

Reply:
i think we can remove the call to `SetLabels` safely.